spec/requests/admin/schools/cohort_2020_spec.rb

```diff
@@ -9,8 +9,8 @@
   let(:cohort_2020) { create(:cohort, start_year: 2020) }
   let(:school_cohort) { create(:school_cohort, :cip, cohort: cohort_2020, school:, core_induction_programme: cip) }
   let!(:participants) { create_list(:ect_participant_profile, 5, school_cohort:) }
-  let(:other_cohort) { create(:cohort) }
-  let(:other_school_cohort) { create(:school_cohort, :cip, cohort: other_cohort, school: school_cohort.school, core_induction_programme: cip) }
+  let(:cohort_2021) { Cohort.find_by(start_year: 2021) || create(:cohort, :current) }
```

Comment:
I don't think we can be in a position where 2021 doesn't exist as it's explicitly created in the `initial_seed.rb`